app/templates/suggestion/index.html

```diff
@@ -1,7 +1,42 @@
 {% extends 'layouts/base.html' %}
-
+<!--should we have a "mark as read" column in the table -->
 {% block content %}
   <div class="ui text container">
     This is a page for suggestions.
+    <table>
+      <thead>
+        <tr>
+          <th>Timestamp</th>
+          <th>Suggestion</th>
+          <th>Contact Info</th>
+          <th>Action</th>
+        </tr>
+      </thead>
+      <tbody>
+        {% for suggestion in suggestions %}
+          <tr>
+            <td>{{ suggestion['timestamp'] }}</td>
```

Comment:
Suggested format: `suggestion.timestamp` instead of `suggestion['timestamp']`. Even though both are valid, the former doesn't require using strings, which are error-prone.